src/makeCalculator.js

```diff
@@ -4,7 +4,32 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add: (num) => result + num,
```

Comment:
The `add` method should update the `result` variable. Consider changing it to `result += num;`.